pkgs/development/python-modules/pycparser/default.nix

```diff
@@ -22,6 +23,10 @@ buildPythonPackage rec {
   nativeCheckInputs = [ unittestCheckHook ];
   disabled = pythonOlder "3.8";
 
+  preCheck = lib.optionalString stdenv.cc.isClang ''
+    substituteInPlace examples/using_gcc_E_libc.py \
+      --replace-fail "'gcc'" "'clang'"
```

Comment:
Maybe `${stdenv.cc.targetPrefix}cc` is better so cross works?

Reply:
Switched and made unconditional.

Reply:
Also built on macos 12 x86_64 successfully.